driver/fixtures/fixtures.go

```diff
@@ -44,6 +44,11 @@ type DockerConfig struct {
 	Image string
 }
 
+func runsInDocker() bool {
```

Comment:
What you wrote here is completely fine, but I would consider making it specific:

```go
func skipInDocker(t *testing.T) {
  t.Helper()
  if _, err := os.Stat("/.dockerenv"); err == nil {
    t.SkipNow()
  }
}
…
skipInDocker(t)
```

It's really up to you, but it's nice to reduce the conditional structure inside the Test functions.

Reply:
Oh. Now I see you do this for both *testing.T and *testing.B, so nevermind.